client/landing/stepper/declarative-flow/test/site-migration-flow.tsx

```diff
@@ -298,7 +298,23 @@ describe( 'Site Migration Flow', () => {
 			} );
 		} );
 
-		it( 'Uses the siteId param fallback', async () => {
+		it( 'redirects the user to the internal import flow if they are comming from the customer home', () => {
```

Comment:
I have two comments here:
 * "Customer Home" refers to `/home/:siteSlug`, but the actual source is `/import/:siteSlug` so I think we should use something like "in-product importer" or "Calypso import page"
 * There is a small typo in "comming" => "coming"
```suggestion
		it( 'redirects the user to the internal import flow if they are coming from the customer home', () => {
```